spellbook/models/tokens/avalanche/tokens_avalanche_nft.sql

```diff
@@ -0,0 +1,1503 @@
+{{ config( alias='nft') }}
+
+SELECT LOWER(contract_address) AS contract_address, name, symbol
+FROM (VALUES
+        ('0xdce4867e34f977619b296646b04bdb334b8f6950', 'Bananaz', 'NANAZ'),

```

Comment:
@soispoke should this new nft metadata file match ethereum, in terms of additional columns for 'standard' and 'category'?

also, is the cast() to string required, similar to ethereum?

Reply:
@jeff-dude I think it should definitely include the 'standard' column. The 'category' one was a bit more specific to a particular case we had on ethereum, but we could add for consistency so it makes it easier to query tokens across chains. The cast() string is not required, we just need to make sure tokens added here are all strings (using single quotes).

Reply:
@0xManny please refer to the following for two extra columns: https://github.com/duneanalytics/abstractions/blob/master/spellbook/models/tokens/ethereum/tokens_ethereum_nft.sql

as mentioned, the cast() logic isn't necessary, just ensure the values are included in single quotes.